cleanup

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 # Delete old "bootstrap" directories that mock leaves behind.
 # (Will no longer be needed after Koji 1.22.)
-find /var/lib/mock -maxdepth 1 -name \*-bootstrap -type d -mtime +2 -exec rm -rf {} +
+find /var/lib/mock -maxdepth 1 -name \*-bootstrap -type d -mtime +0 -exec rm -rf {} +
+#    +0 means _greater_ than 0 days, i.e. only delete stuff older than 24 hours
+# This matches the code implemented in Koji 1.22.
```

Comment:
I kind of wish these would just go away immediately when the builds finish - or some kind of event-based system for cleaning these.  Even 24 hours is a long time if you have lots of builds, i guess.  But i guess if that's how they do it in the future...

Reply:
it's complicated I guess? https://pagure.io/koji/issue/1988

Reply:
Interesting, it looked to me like the bootstrap dirs got cleared out earlier, but the dir itself would be left a bit longer

```
# If rpmlib is installing in this chroot, removing it entirely
# can lead to a world of hurt.
# We remove the rootdir contents but leave the rootdir unless it
# is really old
```


and, from your link,

>@ praiskup There is more complicated policy for that. If build is successful, then after two minutes we delete contents of rootdir. After another 24 hours we delete rootdir itself + mock config. There are some issues with hanging processes and interaction with rpmlib. For failed buildroots there is more complicated policy to have them long enough for debugging. So, it is being done manually outside of mock.

Something like:

```
find /var/lib/mock -maxdepth 1 -name \*-bootstrap -type d -mmin +1 -exec find {} -mindepth 1 -delete \;
````

except maybe not quite because mtime might not be the same as "build success time".

...

There was also some talk about the `--scrub bootstrap` option to mock.


>>@ tkopecek I'm curious if we can rely on mock's clean/scrub behavior totally in koji

>Yes, but not for the version in epel6. When we drop rhel6 support we can drop also this complicated cleaning code.

Which makes me wonder if we could use that with the epel7 version of mock...